Elias/exercise-6.js

```diff
@@ -0,0 +1,53 @@
+let ticketsSum = 6450;
+let ticketCost = 15;
+
+let ticketsSold = ticketsSum / ticketCost;
+
+console.log(ticketsSold);
+
+//answer: 430 tickets were sold!
+
+let sylviasIncomePerWeek = 500;
+let weeksWithinAmonth = 4;
+let monthsWithinAyear = 12;
```

Comment:
The second answer is wrong. There is 52 weeks per year and not 48!
The solution is :
let y = 500 * 52 which the result is 26k. But your result is 24k a year!